dataprofiler/profilers/profile_builder.py

```diff
@@ -1673,7 +1739,7 @@ def diff(  # type: ignore[override]
                 ),
                 "duplicate_row_count": utils.find_diff_of_numbers(
                     self._get_duplicate_row_count(),
-                    other_profile._get_row_is_null_ratio(),
```

Comment:
why didn't this fail any tests?

Reply:
no tests failed cause the values were set to pass not to be correct, as seen below

Reply:
Huge catch